pvsite_datamodel/sqlmodels.py

```diff
@@ -58,11 +60,11 @@ class SiteGroupSQL(Base, CreatedMixin):
 
     # Relationships
     # N-N
-    sites: List["SiteSQL"] = relationship(
+    sites: Mapped["SiteSQL"] = relationship(
```

Comment:
Im worried this is no longer a list?

Reply:
According to my search, the use of `Mapped` for annotating values doesn't change its list-like behaviour at runtime; it still supports typical list operations such as appending and iteration. This change primarily enhances type clarity and tool support, aligning with SQLAlchemy's recommendations for explicit ORM mappings in version 1.4 and beyond.